docs/cli/kops_create.md

```diff
@@ -42,7 +50,12 @@ kops create -f FILENAME
 ### Options
 
 ```
-  -f, --filename stringSlice   Filename to use to create the resource
+  -f, --filename stringSlice    Filename to use to create the resource
+  -m, --model string            Models to apply (separate multiple models with commas) (default "config,proto,cloudup")
+  -o, --out string              Path to write any local output
```

Comment:
This seems overloaded with the `-o` output format options, maybe just use stdout/stderr and let people redirect as desired?